django/librarian/models.py

```diff
@@ -99,25 +99,43 @@ class Meta:
 
     def clean(self):
         self._validate_public_library()
-        # Ensure that there is at most 1 default library
-        if (
-            self.is_default_library
-            and Library.objects.filter(is_default_library=True)
-            .exclude(pk=self.pk)
-            .exists()
-        ):
-            raise ValidationError("There can be only one default library")
+        self._validate_default_library()
+        self._validate_personal_library()
```

Comment:
Add some validation rules: personal library can't be the default library, or a public library.